lib/components/dropdown-item.vue

```diff
@@ -1,31 +1,21 @@
 <template>
     <b-link class="dropdown-item"
             role="menuitem"
```

Comment:
I'm pretty confident this will solve issues, not create them, but I'd love to give it some time in the wild to confirm its stability.